tensorflow/tools/ci_build/windows/cpu/pip/build_tf_windows.sh

```diff
@@ -32,28 +32,11 @@ set -e
 # This script is under <repo_root>/tensorflow/tools/ci_build/windows/cpu/pip/
 # Change into repository root.
 script_dir=$(dirname $0)
-cd ${script_dir%%tensorflow/tools/ci_build/windows/cpu/pip}
+cd ${script_dir%%tensorflow/tools/ci_build/windows/cpu/pip}.
```

Comment:
Oh, the extra dot is for addressing the mentioned problem. We don't want to jump to home directory if we run the script under TF source root.
If we add the extra dot, it would be `cd .`, which stays at the source root.